### ISSUE TITLE
create_simulation_parameter does not reflect change in "measures" parameter of the function

### ISSUE THREAD

**Issue by HuifangYeo** [atoti/atoti#343]
## Steps to reproduce
Initially, I have run the notebook with the below simulation definition (`Best results` in first position followed by `points substitute` in the second position:

```
pointssystem_sim = f1cube.create_parameter_simulation(
    "pointssystem_sim",
    measures={"Best results": 1000.0, "points substitute": None},
    levels=[l["positionText"], l["resultId"]],
)
```
I switched the positions of the measures around but it seems that the values are applied based on the initial setup sequence.
So the values meant for `Best results` is applied to `points substitute` and vice versa. 

![image](https://user-images.githubusercontent.com/35210495/130580228-5339a155-f49b-47b2-8b3b-2fc66c1ed193.png)

[main.zip](https://github.com/atoti/atoti/files/7037586/main.zip)

## Environment

- atoti: 0.6.1
- Python: 3.9.6
- Operating System: win32

**Comment by Skyscimitar** [comment]
The issue seams to be that the order of the map's keys is lost when passing the argument to the underlying java process, for now you can check the order of the created simulation's columns to see the real order of the columns.

I'll fix this behavior for the next release.

**Comment by bloussou** [comment]
Fixed since atoti `0.6.2`
